platforms/hyperledger-besu/charts/node_besu/values.yaml

```diff
@@ -167,3 +167,7 @@ labels:
   service: []
   pvc: []
   deployment: []
+  
+metrics:
```

Comment:
Please provide example values for each k:v pair (check other key:value pairs in this values file)
And I don't think we need the `enabled` field. This could be easily considered by the presence/absence of the metrics port.